plugin/src/main.rs

```diff
@@ -0,0 +1,86 @@
+use clap::Parser;
+use kube::Client;
+use kubectl_plugin::resources;
+use plugin::ExecuteOperation;
+use std::{env, ops::Deref};
+
+pub(crate) mod cli_utils;
+
+#[derive(Parser, Debug)]
+#[clap(name = utils::package_description!(), version = utils::version_info_str!())]
+#[group(skip)]
+struct CliArgs {
+    #[clap(subcommand)]
+    operations: cli_utils::Operations,
+
+    /// Namespace where openebs is installed.
+    /// If unset, defaults to the default namespace in the current context.
+    #[clap(global = true, long, short = 'n')]
+    namespace: Option<String>,
```

Comment:
Why default why not openebs?

Reply:
Its because if its not supplied we will take default namespace set on kubeconfig context. We want to have this behaviour in mayastor plugin also. But that will be breaking change.